readfile.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
+import numpy as np
+
+
+def read_file(file_path):
+
+    return pd.read_csv(file_path, sep=" ", header=None).dropna(axis="columns")
```

Comment:
The separator seems to be 3 spaces or something of that like in the inputs, the dropna handles this specific case but that might not be the best way to do it. Why not just us the multispaced separator?

Reply:
That's a good idea.